examples/sw5/devenv.yaml

```
@@ -3,9 +3,9 @@ imports:
   - kellerkinder
 inputs:
   nixpkgs:
-    url: github:NixOS/nixpkgs/nixos-unstable
+    url: github:NixOS/nixpkgs?ref=23.11
```

```suggestion
    url: github:NixOS/nixpkgs/nixos-23.11
```
Wondering why the tag and the branch for 23.11 differ so much. Any ideas?

---

This change was about "using the release". The branch contains updates which get backported, mainly security updates.
It might be worth trying using this? 🤔 
https://status.nixos.org/

---

In general, I would be in favour of using the branch `nixos-23.11` (due to updates, patches, …). 
Unfortunately, it just noticed, that we need the branch `nixos-23.05` for `nodejs_16`.  Should we add another example for this?